committee-code-of-conduct/election.md

```diff
@@ -0,0 +1,78 @@
+# Kubernetes Code of Conduct Committee Elections
+
+This document outlines the process, for Code of Conduct Committee (CoCC) elections.
+
+### Length of Terms
+
+* The CoCC consists of 5 members. In the first election, the top 3 voted people will be appointed for 2 year terms and the other 2 members will be appointed for a 1 year term.
+* In all subsequent elections the term will be for 2 years.
+* In alternating years either 2 or 3 seats will be up for election as terms expire.
+
+### Eligibility for voting
+
+* Voting will be done by members of the steering committee.
+
+### Eligibility for candidacy
+
+* Candidate commits to putting the interests of the community above the interests of their employer for all Code of Conduct Committee activities.
+* Is generally a responsible human
+* Does not have to be part of the Kubernetes or CNCF community
+* Characteristics which would be nice in any nominations:
+  * Previous experience on an Ethics Committee or Code of Conduct Committee is appreciated
+  * Has demonstrated integrity, professionalism, and positive influence within the community is appreciated
+  * Experience with the tools which we use to communicate (Zoom, Slack, GitHub, etc.) within the Kubernetes community is appreciated
+
+### Election process
+
+The steering committee will announce a request for nominations some time in July from any member of the following:
+* steering committee
+* sig chair
+* sig lead
+
+The committee will accept nominations for at least 3 weeks. After the announced nomination
+period the steering committee will hold a private ballot election using time-limited
+[Condorcet] ranking use the [Schulze method] to break ties. The steering committee
+will publicly announce the results of the election.
+
+### Maximal representation
+
+Since we require commitment that all members of the committee put the interests
+of the community above the interest of their employer, or [recuse themselves] from all
+relevant proceedings,  while performing all duties related to the CoCC we do not
+have a maximal corporate representation limit.
+
+### Election Cycles
+
+Election cycles are scheduled such that roughly half of the seats come up for
+re-election each year for purposes of continuity.  The exact number of seats
+alternates between 2 and 3. The elections by the steering committee should be
+held and announced before the previous appointments expire (typically in Aug).
+
+### Emeritus Term
+
+There is no concept of an "Emeritus Term" with the CoCC. When a member vacates
+their seat for any reason they will be removed from all conversations, e-mail
+lists, historical data, and any other activities related to the CoCC.
+
+### Vacancies
+
+In the event of a resignation or other loss of an elected committee
+member, the candidate with the next most votes from the previous election will
+be offered the seat.  This process will continue until the seat is filled.
```

Comment:
What are the chances this results in a member with a very small number of votes for being granted the position?

Reply:
the alternative being steering holds an ad-hoc vote?

Reply:
Or a snap election?  Or control the depth of the election stack (i.e. first alternate / second alternate)?

Reply:
i think a special run-off election is probably the best outcome.

Reply:
I see 2 votes for a snap election instead of taking the next on the list (what steering does). If steering wants to potentially do more elections I'm happier with that too.